lib/pygments/popen.rb

```diff
@@ -215,7 +215,7 @@ def mentos(method, args=[], kwargs={}, original_code=nil)
 
       begin
         # Timeout requests that take too long.
-        timeout_time = ENV["MENTOS_TIMEOUT"] || 8
+	timeout_time = Integer(ENV["MENTOS_TIMEOUT"]) rescue 8
```

Comment:
Fixed. Lesson learned: I'm an Emacs user, where I have tabs turned off, but was editing this file repeatedly with Vim.